var/spack/repos/builtin/packages/opencv/package.py

```diff
@@ -93,44 +121,97 @@ class Opencv(CMakePackage):
     depends_on('py-numpy', when='+python', type=('build', 'run'))
     depends_on('protobuf@3.1.0', when='@3.3.0: +dnn')
 
+    depends_on('ffmpeg', when='+videoio')
+
     extends('python', when='+python')
 
     def cmake_args(self):
         spec = self.spec
 
+        # Standard variants
         args = [
             '-DBUILD_SHARED_LIBS:BOOL={0}'.format((
                 'ON' if '+shared' in spec else 'OFF')),
             '-DENABLE_PRECOMPILED_HEADERS:BOOL=OFF',
-            '-DWITH_IPP:BOOL={0}'.format((
-                'ON' if '+ipp' in spec else 'OFF')),
-            '-DWITH_CUDA:BOOL={0}'.format((
-                'ON' if '+cuda' in spec else 'OFF')),
-            '-DWITH_QT:BOOL={0}'.format((
-                'ON' if '+qt' in spec else 'OFF')),
-            '-DWITH_VTK:BOOL={0}'.format((
-                'ON' if '+vtk' in spec else 'OFF')),
-            '-DBUILD_opencv_java:BOOL={0}'.format((
-                'ON' if '+java' in spec else 'OFF')),
+            '-DWITH_LAPACK={0}'.format((
+                'ON' if '+lapack' in spec else 'OFF')),
+            '-DENABLE_POWERPC={0}'.format((
+                'ON' if '+powerpc' in spec else 'OFF')),
+            '-DENABLE_VSX={0}'.format((
+                'ON' if '+vsx' in spec else 'OFF')),
+            '-DENABLE_FAST_MATH={}'.format((
```

Comment:
@bvanessen the `{}`-style formatting is not compatible with python 2.6, update it to `{0}`